### PR TITLE
Added react-intl and testing trivial example

### PR DESCRIPTION
**Pull request by k88hudson**
I figured I would put up the JS strategy in progress for y'all to comment if you have ideas.

So far, `react-intl` seems like a pretty good solution to me; I'm planning on passing in the locale and strings from Android via the bridge.

### CODE REVIEW

File: www_src/lib/render.jsx [hunk 1]
```diff
@@ -3,7 +3,18 @@ var Spindicator = require('../components/spindicator/spindicator.jsx');
 var ModalConfirm = require('../components/modal-confirm/modal-confirm.jsx');
 var ModalSwitch = require('../components/modal-switch/modal-switch.jsx');
 
+var intlData = {
```
Comment (k88hudson) [inline www_src/lib/render.jsx]: this will be from android

**Comment by k88hudson** [comment]
cc @thisandagain / @pomax / @gvn

**Comment by gvn** [comment]
Seems like a good approach. When running on a desktop browser will localized strings just be empty?

**Comment by alicoding** [comment]
So, I'm trying to understand what would be the benefit for us using XML for our string resource here? Is it just so we can use it in both React and on Android side? I really like the idea of us using ICU format for translation since it makes it really easy to maintain and readable. Also, it is supported by default with React-Intl.

What do you think about separating them totally in this case? For example:

Continue using XML in Webmaker-android and take of all the webmaker-core strings. Use JSON for webmaker-core - I like the idea of sharing, but having to build the file and share one giant resource file is hard to maintain.

Thought? @Pomax @thisandagain @k88hudson @gvn

**Comment by toolness** [comment]
<a href="https://github.com/mozilla/webmaker-core/issues/193"></a>
This issue has been moved to https://github.com/mozilla/webmaker-core/issues/193.
Please don't comment on it here.

**Comment by toolness** [comment]
Sorry, my migration script for https://github.com/mozilla/webmaker-core/issues/23 accidentally migrated this, even though it wasn't supposed to. Just undid it!

**Pull request closed without merging**